### PR TITLE
Update `firebase.json` with `/to/spm` for Swift Package Manager page

### PR DESCRIPTION
**Pull request by khanhnwin**
We're releasing a short video that promotes the new Swift Package Manager docs. We'd like a shorter URL to send folks to the docs.

### CODE REVIEW

**Comment by khanhnwin** [comment]
cc @loic-sharma

File: firebase.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
       { "source": "/sdk-archive", "destination": "/release/archive", "type": 301 },
       { "source": "/setup-:os*", "destination": "/get-started/install/:os*", "type": 301 },
       { "source": "/setup", "destination": "/get-started/install", "type": 301 },
+      { "source": "/spm", "destination": "/packages-and-plugins/swift-package-manager", "type": 301 },
```
Comment (parlough) [inline firebase.json:87]: What will this be used for? I'd prefer to use a `/to/spm` [tooling redirect](https://github.com/flutter/flutter/blob/master/docs/contributing/use-reliable-links.md) if possible.

Comment (khanhnwin) [reply firebase.json:87]: It'll be used in a YouTube Short video. Good to use whichever format you prefer!

**Review by parlough — APPROVED** [review]
Thanks @khanhnwin!

I've updated the redirect to use `/to/spm`, so after this lands `https://flutter.dev/to/spm` should work :)

**Comment by khanhnwin** [comment]
Thank you @parlough!

**Pull request merged**